examples/RDMSerialRecv/RDMSerialRecv.ino

```diff
@@ -31,6 +31,9 @@
 // 25.05.2017 Stefan Krupop: Add support for sensors
 // 21.08.2018 improvements and typo by Peter Newman
 // 31.10.2018 Remove unnecessary #include <EEPROM.h> by Graham Hanson
+// 04.06.2023 Tim Nijssen: Add support for device-specific parameters
+// 05.06.2023 Tim Nijssen: integrate sensors example
+// 05.06.2023 Tim Nijssen: Add example for device-specific parameters

```

Comment:
I genuinely don't know, and a lot of other stuff has been duplicated, but should these comments be in this ino file given they aren't relevant to it?